.github/workflows/release-nakama.yaml

```
@@ -52,6 +52,14 @@ jobs:
             type=semver,pattern={{version}},value=${{ steps.semver.outputs.tag }}
             type=semver,pattern={{major}}.{{minor}},value=${{ steps.semver.outputs.tag }}
             type=sha
+      - name: Prepare arm64 tags
+        id: arm64_tags
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2129:style:3:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)